import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...), get_code_authorization_token(...) or get_pkce_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.ExternalContactsApi();
contact_id = 'contact_id_example' # str | ExternalContact ID
body = PureCloudPlatformClientV2.IdentifierClaimRequest() # IdentifierClaimRequest | ClaimRequest

try:
    # Claim or release identifiers for a contact
    api_response = api_instance.patch_externalcontacts_contact_identifiers(contact_id, body)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling ExternalContactsApi->patch_externalcontacts_contact_identifiers: %s\n" % e)